docs/guides/allocation-limits.mdx

```diff
@@ -5,12 +5,13 @@ platform: cloud
 ---
 # Set instance allocation limits
 
-An account might have allotted time on the premium plan. Because each account can have many premium plan instances, setting allocation limits helps you control which instances are allowed to consume that time. Note that if the allocation is not set as a limit, the instance can use more than its alloted time.
+An account might have allotted time on the Premium or Flex plan. Because each account can have many instances on a plan, setting allocation limits helps you control which instances are allowed to consume that time. Note that if the allocation is not set as a limit, the instance can use more than its alloted time.
```

Comment:
```suggestion
An account might have allotted time on the Premium or Flex plan. Because each account can have many instances on a plan, setting allocation limits helps you control which instances are allowed to consume that time. Note that if the allocation is not set as a limit, the instance can use more than its allotted time.
```